avaframe/com1DFA/com1DFA.py

```diff
@@ -266,21 +269,50 @@ def prepareRelase(cfg, rel, inputSimLines):
             message = 'No secondary release file found'
             log.error(message)
             raise FileNotFoundError(message)
-        secondaryReleaseLine = inputSimLines['secondaryReleaseLine']
-        for k in range(len(secondaryReleaseLine['d0'])):
-            if secondaryReleaseLine['d0'][k] == 'None':
-                secondaryReleaseLine['d0'][k] = cfg['GENERAL'].getfloat('secondaryRelTh')
-            else:
-                secondaryReleaseLine['d0'][k] = float(secondaryReleaseLine['d0'][k])
+        secondaryReleaseLine = setThickness(cfg, inputSimLines['secondaryReleaseLine'], 'useRelThFromIni', 'secondaryRelTh')
     else:
         inputSimLines['entResInfo']['flagSecondaryRelease'] = 'No'
         secondaryReleaseLine = None
-
     inputSimLines['secondaryReleaseLine'] = secondaryReleaseLine
 
+    if entResInfo['flagEnt'] == 'Yes':
+        # set entrainment thickness
+        entLine = setThickness(cfg, inputSimLines['entLine'], 'useEntThFromIni', 'entTh')
+        inputSimLines['entLine'] = entLine
+
     return relName, inputSimLines, badName
 
 
+def setThickness(cfg, lineTh, flagTh, typeTh):
+    """ set thickness in line dictionary for release area, entrainment area
+
+    Parameters
+    -----------
+    lineTh: dict
+        dictionary with info on line (e.g. release area line)
+    flagTh: bool
```

Comment:
See issue #86 : we are trying not to use flagSOMETHING, so maybe change this to useThFromIni

Reply:
changed to useThFromIni